docs/internet-identity-spec.adoc

```diff
@@ -342,8 +342,8 @@ Therefore the Internet Identity Canister intentionally does not implement `canis
 
 The primary data structure used by the backend is a map from Identity Anchor to the list of user devices.
 Device lists are stored directly in canister stable memory.
-The total amount of storage for is limited to 510 bytes per user.
-With the stable memory size of 4GiB we can store around 8 * 10^6 user records in a single canister.
+The total amount of storage for is limited to 2k bytes per user.
```

Comment:
Mentioned by @kritzcreek but I didn't check the math.

Reply:
It's 2kb as specified here: https://github.com/dfinity/internet-identity/blob/5f5bd98e44f89ebd321a5c3b3d5ee36d7cd4a799/src/internet_identity/src/storage.rs#L12

Reply:
Updated to 2KiB